### PR TITLE
Cknieling/update doc

### PR DESCRIPTION
**Pull request by chr11115**
- Merge all readme files into one
- Extend user manual:
  - Add section to introduce `pldAssign` and `pldGenerate` nodes
  - Update all screenshots
  - Add section for working with multiple assign nodes
  - Add section for working with HDA material
  - Provide RPK needed for the examples in doc/data
- Add new abbreviated Quick Start guide
- Update requirements
- Rename changelog to release notes

Note: currently there is a section where we want the user to check the release notes for the software requirements. This is currently not maintained and should be added to the release notes for the next and all upcoming releases like in puma and serlio

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +231,234 @@ It can be useful to put RPKs into an `rpk` sub-directory of your current Houdini
...
+
+#### Linux
+1. Ensure GCC 9.3 is active.
+1. `cd` into your Palladio git repository
+1. `mkdir -p build/release && cd build/release`
+1. `cmake -DCMAKE_BUILD_TYPE=Release ../../src`
+1. `make install` (the plugin will be installed into your `~/houdini19.5/dso` directory)
+
+#### Windows
+1. Open a MSVC 14.2 x64 shell (Visual Studio 2019) and `cd` to the Palladio git repository
```
Comment (mistafunk) [inline README.md:285]:
```suggestion
1. Open a MSVC 14.27 x64 shell (Visual Studio 2019) and `cd` to the Palladio git repository
```

File: README.md [hunk 2]
```diff
@@ -70,6 +231,234 @@ It can be useful to put RPKs into an `rpk` sub-directory of your current Houdini
...
+1. `cmake -G "NMake Makefiles" -DCMAKE_BUILD_TYPE=Release ../../src`
+1. `nmake install` (the plugin will be installed into your `%USERPROFILE%/Documents/houdini19.5/dso` directory)
+
+### Running Palladio
+See [Quick Start](#quick-start) how to launch Houdini with Palladio.
+
+### Building and Running Unit Tests
+
+#### Linux
+1. Ensure GCC 6.3 is active.
```
Comment (mistafunk) [inline README.md:297]:
```suggestion
1. Ensure GCC 9.3 is active.
```

File: README.md [hunk 3]
```diff
@@ -70,6 +231,234 @@ It can be useful to put RPKs into an `rpk` sub-directory of your current Houdini
...
+1. `cd` into your Palladio git repository
+1. `mkdir -p build/relTest && cd build/relTest`
+1. `cmake -DCMAKE_BUILD_TYPE=Release -DPLD_TEST=1 ../../src`
+1. `make palladio_test`
+1. run `bin/palladio_test`
+
+#### Windows
+1. Open a MSVC 14.2 x64 shell (Visual Studio 2019) and `cd` to the Palladio git repository
```
Comment (mistafunk) [inline README.md:305]:
```suggestion
1. Open a MSVC 14.27 x64 shell (Visual Studio 2019) and `cd` to the Palladio git repository
```

File: README.md [hunk 4]
```diff
@@ -70,6 +231,234 @@ It can be useful to put RPKs into an `rpk` sub-directory of your current Houdini
...
+1. `mkdir build/relTest`
+1. `cd build/relTest`
+1. `cmake -G "NMake Makefiles" -DCMAKE_BUILD_TYPE=Release -DPLD_TEST=1 ../../src`
+1. `nmake palladio_test`
+1. ensure that the `bin` subdirectory of your Houdini installation is in the `PATH`
```
Comment (mistafunk) [inline README.md:310]: this is a leftover from when the tests still depended on Houdini, please remove this line

File: README.md [hunk 5]
```diff
@@ -24,6 +36,155 @@ External documentation:
...
+### Overriding Rule Attributes
+
+In the previous section we've used the default value for the `height` attribute. Let's use Houdini tools to modify the rule attribute prior to execution.
+
+#### Using Parameter Editor
+
+1. Make sure the `pldAssign` is evaluated by making the connected `pldGenerate` the active render node
+1. Edit the rule attribute in the parameter editor by changing the value in the generated handle (a rule attribute can be reset by calling `revert to Defaults` or pressing `ctrl + MMB` on the rule name)
+
+![](doc/img/attribute03.png)
```
Comment (mistafunk) [inline README.md:108]: the images in this section are not aligned with the enumeration lists, try to align them

**Review by mistafunk — CHANGES_REQUESTED** [review]
Overall nice work! Found a bunch of text issues and left-overs

File: README.md [hunk 6]
```diff
@@ -6,15 +6,27 @@ Palladio requires so-called rule packages (RPK) as input, which are authored in
...
+## Quick Start
+
+1. Download the latest palladio binary from the [release page](https://github.com/esri/palladio/releases).
+1. Extract the archive into your `<home>\Documents\houdiniX.Y` (Windows) or `<home>/houdiniX.Y` (Linux) directory, so that the files end up in the corresponding `config`, `dso` and `packages` sub-directories.
+1. Start Houdini and Palladio will be loaded automatically. 
+1. In Houdini create an initial shape (i.e. a `grid` node with 2 cols/rows).
+1. Open the geometry node in the network editor and connect the mesh output to a new `pldAssign` node.
+1. Create a new `pldGenerate` node and connect the input to the ouput of the previous `pldAssign` node.
+1. Make the `pldGenerate` node the active render node.
+1. In CityEngine, download e.g. [Tutorial 9](https://doc.arcgis.com/en/cityengine/latest/tutorials/tutorial-9-advanced-shape-grammar.htm) and export the "Parthenon" CGA rule to a RPK (see [Creating a Rule Package](https://doc.arcgis.com/en/cityengine/latest/help/help-rule-package.htm)).
```
[the remark this answers is not in the document]
Comment (chr11115) [reply README.md]: added parthenon rpk

**Comment by mistafunk** [comment]
I've added two doc cleanups, please counter review and merge if happy.

**Review by mistafunk — APPROVED** [review]

**Comment by chr11115** [comment]
LGTM, I'll merge

**Comment by chr11115** [comment]
LGTM, I'll merge

**Pull request merged**